src/components/link/link.helper.js

```diff
@@ -68,10 +68,10 @@ function getRadiusStrategy(type) {
  * @returns {string} the path definition for the requested link
  * @memberof Link/helper
  */
-function buildLinkPathDefinition({ source = {}, target = {} }, type = LINE_TYPES.STRAIGHT) {
+function buildLinkPathDefinition({ source = {}, target = {}, linktype }, type = LINE_TYPES.STRAIGHT) {
     const { x: sx, y: sy } = source;
     const { x: tx, y: ty } = target;
-    const validType = LINE_TYPES[type] || LINE_TYPES.STRAIGHT;
+    const validType = LINE_TYPES[linktype] || LINE_TYPES[type];
```

Comment:
Same comment here, in fact, no need to update this function at all, it should work just fine with the previous implementation.